### ISSUE TITLE
panics on termux

### ISSUE THREAD

**Issue by phanirithvij** [amodm/webbrowser-rs#53]
When I try to run a very simple minimal reproducible [example](https://github.com/phanirithvij/sample_webbrowser/blob/main/src/main.rs)

```rust
use webbrowser;

fn main() {
    if webbrowser::open("http://github.com").is_ok() {
        println!("hello, github.com");
    }
}
```

With `webbrowser = "0.8.1"` as the only dependency

```shell
# can be run on termux (rust required, `pkg install rust`)
# cargo install sample_webbrowser --git https://github.com/phanirithvij/sample_webbrowser && RUST_BACKTRACE=full sample_webbrowser
```

The output is
```py
thread 'main' panicked at 'android context was not initialized', /data/data/com.termux/files/home/.cargo/registry/src/github.com-1ecc6299db9ec823/ndk-context-0.1.1/src/lib.rs:72:30
```

**Comment by amodm** [comment]
Based on [this stackoverflow answer](https://stackoverflow.com/questions/58595693/get-or-create-a-context-from-within-an-android-java-process), it seems that Android APIs are not available to programs running inside Termux.

In this scenario, it'd be best to detect the Termux environment and solve for it separately, as against treating it as regular Android env. I'll search online to see what mechanisms others have used to detect Termux env, but if you have any ideas, I'd appreciate the help greatly.

**Comment by amodm** [comment]
As of now, `TERMUX_VERSION` env var seems more appropriate, so I'll be proceeding with that as a way to detect the termux env in Android, unless you have better ideas.

**Comment by amodm** [comment]
@phanirithvij, please give the [android-termux](https://github.com/amodm/webbrowser-rs/tree/android-termux) branch a shot. My local testing seems to show that this works, but I'll await your confirmation for a while before merging this.

**Comment by amodm** [comment]
@phanirithvij in the absence of any confirmation from you, I'll be merging #54 on the basis of my local testing alone, by tomorrow. So if possible, please do give it a test run.

**Comment by amodm** [comment]
This is now released as [v0.8.2](https://github.com/amodm/webbrowser-rs/releases/tag/v0.8.2)

**Comment by phanirithvij** [comment]
Works! Thanks for the quick fix. Sorry for responding late.
